package.json

```diff
@@ -7,7 +7,7 @@
     "clean": "rm -rf dist",
     "start": "node dist/main.js -p 8080 --log-level=debug",
     "serve_local": "npm run build && nodemon dist/main.js -- --dev --log-level=silly --port=8040",
-    "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js' --require @babel/register --require test_setup.js",
+    "test": "NODE_ENV=test mocha --exit --reporter spec --require @babel/register --require test_setup.js 'src/*_test.js'",
```

Comment:
It was bothering me that this was just hovering in the middle of the command so I moved it to the end.